modules/gapi/perf/gpu/gapi_core_perf_tests_gpu.cpp

```diff
@@ -73,25 +79,28 @@ INSTANTIATE_TEST_CASE_P(DivPerfTestGPU, DivPerfTest,
                                 Values(cv::compile_args(CORE_GPU))));
 
 INSTANTIATE_TEST_CASE_P(DivCPerfTestGPU, DivCPerfTest,
-                        Combine(Values( szSmall128, szVGA, sz720p, sz1080p ),
+                        Combine(Values(AbsExact().to_compare_f()),
+                                Values( szSmall128, szVGA, sz720p, sz1080p ),
                                 Values( CV_8UC1, CV_8UC3, CV_16UC1, CV_16SC1, CV_32FC1 ),
                                 Values( -1, CV_8U, CV_16U, CV_32F ),
                                 Values(cv::compile_args(CORE_GPU))));
-
-INSTANTIATE_TEST_CASE_P(DivRCPerfTestGPU, DivRCPerfTest,
-                        Combine(Values(Tolerance_FloatRel_IntAbs(1e-5, 2).to_compare_f()),
+//TODO: DivRC test doesn't work with (CV_16SC1, -1)
+INSTANTIATE_TEST_CASE_P(DISABLED_DivRCPerfTestGPU, DivRCPerfTest,
```

Comment:
I don't think it should be disabled completely.. maybe just comment `CV_16SC1`?

Reply:
Need to remove only CV_16SC1 from this instantiation.